payments-core/src/main/java/com/stripe/android/paymentsheet/PaymentSheetActivity.kt

```diff
@@ -177,12 +177,12 @@ internal class PaymentSheetActivity : BaseSheetActivity<PaymentSheetResult>() {
 
     override fun onBackPressed() {
         if (supportFragmentManager.backStackEntryCount > 0) {
-            updateErrorMessage(null)
+            updateErrorMessage()
```

Comment:
Just a stylistic thing here, I realize you can use default arguments, but without the null parameter passed in here, it is hard to realize that the side-effect of this function is that the error message will be cleared.